ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -23,6 +23,17 @@ images:
 - dockerfile_path: dev/Dockerfile
   from: base
   to: cincinnati-graph-data-container
+operator:
+  bundles:
+  - as: cincinnati-bundle-container
+    context_dir: .
```

Comment:
No need to have this line, `.` is the default

Reply:
Dropped. Thanks!